HelpSource/Classes/LevelComp.schelp

```diff
@@ -0,0 +1,154 @@
+TITLE:: LevelComp
+summary:: Level compensation logic for Splay and SplayAz
+categories:: Panning
+related:: Classes/Splay, Classes/SplayAz, Guides/Level_Compensation
+
+DESCRIPTION::
+Mixing and Panning multichannel signals often requires link::Guides/Level_Compensation::.
+LevelComp provides a logic for this that is used internally in the link::Classes/Splay:: and link::Classes/SplayAz:: UGens.
+Using the code::levelComp:: argument of code::Splay/Az::, it calculates a level compensation factor applied to the input signals, depending on the number of output channels.
```

Comment:
depending on the number of *input* channels

Reply:
yes right